caddyhttp/push/handler.go

```diff
@@ -58,26 +58,41 @@ outer:
 }
 
 func (h Middleware) servePreloadLinks(pusher http.Pusher, headers http.Header, links []string) {
+outer:
 	for _, link := range links {
-		parts := strings.Split(link, ";")
+		resources := strings.Split(link, ",")
 
-		if link == "" || strings.HasSuffix(link, "nopush") {
-			continue
-		}
+		for _, resource := range resources {
+			parts := strings.Split(resource, ";")
 
-		target := strings.TrimSuffix(strings.TrimPrefix(parts[0], "<"), ">")
+			if link == "" || strings.HasSuffix(resource, "nopush") {
```

Comment:
Shouldn't `nopush` be checked for every item in `parts`?
Maybe add a `nopush` link to the unit test.